packages/babel-polyfill/src/index.js

```diff
@@ -1,7 +1,10 @@
 if (global._babelPolyfill) {
-  throw new Error("only one instance of @babel/polyfill is allowed");
+  throw new Error(
+    "@babel/polyfill is loaded more than once on this page. This is probably not desirable/intended and may have consequences if different versions of the polyfills are applied sequentially. If you do need to load the polyfill more than once, use @babel/polyfill/noConflict instead to bypass the error.",
+  );
 }
+
 global._babelPolyfill = true;
 
-import "core-js/shim";
-import "regenerator-runtime/runtime";
+require("core-js/shim");
```

Comment:
I changed these to `require` since `import` by spec has to be at the top of the file, right?

Reply:
How do you feel about ordering? Most frameworks would do this logic as essentially